proxy/core/src/main/resources/META-INF/native-image/io.cloudstate/cloudstate-proxy-core/akka-cluster-reflect-config.json

```diff
@@ -1,4 +1,9 @@
 [
+  {
```

Comment:
It would be good to record the reason for this (which we can do once it's in HOCON).

The reason is that, otherwise, native-image considers parts of ClusterActorRefProvider method call graph as unreachable:

```
com.oracle.svm.core.jdk.UnsupportedFeatureError: Code that was considered unreachable by closed-world analysis was reached
	at com.oracle.svm.core.util.VMError.unsupportedFeature(VMError.java:101)
	at com.oracle.svm.core.snippets.SnippetRuntime.unsupportedFeature(SnippetRuntime.java:173)
	at akka.remote.RemoteActorRefProvider.unregisterTempActor(RemoteActorRefProvider.scala:182)
	at akka.pattern.PromiseActorRef.stop(AskSupport.scala:630)
  | => tat akka.pattern.PromiseActorRef$.$anonfun$apply$2(AskSupport.scala:673)
	at akka.pattern.PromiseActorRef$.$anonfun$apply$2$adapted(AskSupport.scala:672)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at io.cloudstate.internal.svm.Target_akka_actor_ActorSystemImpl$1.execute(Substitutions.java:61)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at akka.pattern.PromiseActorRef.$bang(AskSupport.scala:581)
	at io.cloudstate.proxy.eventsourced.EventSourcedEntity$$anonfun$receiveCommand$1.applyOrElse(EventSourcedEntity.scala:260)
	at akka.actor.Actor.aroundReceive(Actor.scala:539)
	at akka.actor.Actor.aroundReceive$(Actor.scala:537)
	at io.cloudstate.proxy.eventsourced.EventSourcedEntity.akka$persistence$Eventsourced$$super$aroundReceive(EventSourcedEntity.scala:142)
	at akka.persistence.Eventsourced$$anon$4.stateReceive(Eventsourced.scala:899)
	at akka.persistence.Eventsourced.aroundReceive(Eventsourced.scala:250)
	at akka.persistence.Eventsourced.aroundReceive$(Eventsourced.scala:249)
	at io.cloudstate.proxy.eventsourced.EventSourcedEntity.aroundReceive(EventSourcedEntity.scala:142)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:612)
	at akka.actor.ActorCell.invoke(ActorCell.scala:581)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:268)
	at akka.dispatch.Mailbox.run(Mailbox.scala:229)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:241)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:527)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:193)

```

Reply:
@dwijnand I thought about that, but then on the other hand, every single config setting has its own reason for be included.

Reply:
Right, and I think it's good to record those reasons, at some level of granularity.  A bit like how people record the reasons for mima exceptions, or how Seth records the reasons behind the adaptations in the community build.